qa/suites/rados/monthrash/workloads/rados_mon_workunits.yaml

```diff
@@ -6,6 +6,7 @@ overrides:
     - \(PG_
     - \(MON_DOWN\)
     - \(AUTH_BAD_CAPS\)
+    - \(POOL_NO_REDUNDANCY\)
```

Comment:
I think a safer path is to add
```
mon_warn_on_pool_no_redundancy = false
```
to qa/tasks/ceph.conf.template's [global] section.

Reply:
I have added the option to the template and removed the whitelists.